docs/architecture/adr-009.md

```diff
@@ -0,0 +1,96 @@
+# ADR 009: Adding extensibility to bridge methods of CRC-21 standard
+
+## Changelog
+* 2022-08-02: first draft
+
+## Context
+
+Following the acceptance of [ADR-008](./adr-008.md), a revision to the token contract standard named CRC-21 is introduced. For existing tokens to enjoy the new bridge capabilities, they need to upgrade or migrate their existing contracts to the CRC-21 standard.
+
+On the other hand, there are some foreseeable bridge changes to be introduced in the coming versions. For example IBC relay fee.
+
+This proposal aims to add certain extensibility to the bridge methods of CRC-21 standard to reduce the need for multiple migrations of existing tokens to support new bridge features in the future. This proposal will also provide guidelines for future bridge updates to leverage the proposed standard.
+
+## Decision
+
+### __CronosSendToChainV2 Event
+
+#### Changes
+
+- Introduce event `__CronosSendToChainV2`, successor of event `__CronosSendToChain`;
```

Comment:
Since CronosSendToChain is not release yet, we could simply modify the existing event rather to introduce a new one